indico/modules/events/persons/forms.py

```diff
@@ -20,6 +20,9 @@ class ManagePersonListsForm(IndicoForm):
     default_search_external = BooleanField(_('Include users with no Indico account by default'), widget=SwitchWidget(),
                                            description=_('If enabled, searching people for speakers/authors will '
                                                          'include those with no Indico account by default.'))
+    show_title = BooleanField(_('Show title'), widget=SwitchWidget(),
```

Comment:
"Show titles" would be better. Singular "title" is more likely to be understood as something like the event title.